gov2/s3/GeneratePresignedURL/README.md

```diff
@@ -0,0 +1,10 @@
+### GeneratePresignedURLv2.go
+
+This example retrieves a presigned URL for an Amazon S3 bucket object.
```

Comment:
Spell out "Amazon Simple Storage Service (Amazon S3)" on first mention in a code file.

Reply:
Done